codebrag-service/src/main/scala/com/softwaremill/codebrag/repository/GitSvnRepository.scala

```diff
@@ -7,6 +7,7 @@ import com.softwaremill.codebrag.repository.config.{RepoData, UserPassCredential
 
 class GitSvnRepository(val repoData: RepoData) extends Repository with RepositoryAutoBuilder with GitSvnBranchesModel {
 
+  private val CommandBaseReset = "git reset --hard"
```

Comment:
Indeed, I tried it but it wasn't working so I ended up this way. I thought it wasn't really problematic since it is a 'restore' command for the repo. What do you think?